lib/managers/collaborations.js

```diff
@@ -146,6 +146,17 @@ Collaborations.prototype.create = function(accessibleBy, itemID, role, options,
 			role: role
 		}
 	};
+
+	if (options.can_view_path) {
+		params.body.can_view_path = options.can_view_path;
+	}
+
+	if (options.notify) {
+		params.qs = {
+			notify: options.notify
+		};
+	}
```

Comment:
For both of these parameters, a value of `false` will NOT be passed (since it fails the if check).  You'll need to do e.g. a `typeof options.notify === 'boolean'` check instead of simple loose truthiness.